Kudu.Core/app.config

```diff
@@ -1,19 +1,23 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why is this file showing as a full-fill diff? Maybe some unwanted formatting issue?